src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -367,6 +367,7 @@ describe('ListOfPublishedProfiles', () => {
 
       // Clicking elsewhere should make the successful message disappear.
       fireFullClick((window: any));
+      await retrieveProfileData(profileToken);
```

Comment:
FYI I just removed this line too. I've checked and tests seem to pass even without it.

Reply:
How strange. I couldn't get the test to pass without it!

Reply:
Maybe that's thanks to my other changes then!